### PR TITLE
Add draft shepherd writeup for base-drafts

### PR DESCRIPTION
**Pull request by larseggert**

### CODE REVIEW

File: writeups/base-drafts.md [hunk 1]
```diff
@@ -0,0 +1,314 @@
...
+## 14. Normative References
+<!--
+(14) Are there normative references to documents that are not ready for
+advancement or are otherwise in an unclear state? If such normative references
+exist, what is the plan for their completion?
+-->
+
+The document set contains the following normative references to I-Ds:
+
+* draft-ietf-tsvwg-datagram-plpmtud
```
Comment (martinthomson) [inline writeups/base-drafts.md:231]: This was published as RFC 8899.  We should fix the reference.

File: writeups/base-drafts.md [hunk 2]
```diff
@@ -0,0 +1,314 @@
...
+
+
+## 19. Validation of Formal Language Parts
+<!--
+(19) Describe reviews and automated checks performed by the Document Shepherd to
+validate sections of the document written in a formal language, such as XML
+code, BNF rules, MIB definitions, YANG modules, etc.
+-->
+
+No formal code exists in the document set. draft-ietf-quic-recovery contains
```
Comment (martinthomson) [inline writeups/base-drafts.md:298]: transport and qpack both include pseudocode in appendices.

**Review by martinthomson — APPROVED** [review]
SHIP IT!

I trust that the other shepherds will review this before it is sent.  Merge it before then as you like.

Do you want to tweak CODEOWNERS?

**Comment by larseggert** [comment]
@martinthomson what should I tweak in CODEOWNERS?

**Comment by martinthomson** [comment]
Oh, I see that you already list chairs as owning everything...  No action required (I would not have included that).

File: writeups/base-drafts.md [hunk 3]
```diff
@@ -0,0 +1,312 @@
...
+<!--
+(18) List any new IANA registries that require Expert Review for future
+allocations. Provide any public guidance that the IESG would find useful in
+selecting the IANA Experts for these new registries.
+-->
+
+The document set defines several IANA registries that offer “Provisional
+Registrations” and require Expert review. The IESG should select subject matter
+experts for these registration types; candidates include the document editors
+and the individuals named as contributors in the acknowledgment sections.
```
Comment (LPardue) [inline writeups/base-drafts.md:286]:
```suggestion
The document set defines several IANA registries that offer “Provisional
Registrations” and "Permanent Registration, which both require Expert review.
The IESG should select subject matter experts for these registration types;
candidates include the document editors and the individuals named as
contributors in the acknowledgment sections.
```

**Review by LPardue — APPROVED** [review]

**Review by janaiyengar — APPROVED** [review]
W00t!

File: writeups/base-drafts.md [hunk 4]
```diff
@@ -0,0 +1,312 @@
...
+
+## 18. New "Expert Review" Registries
+<!--
+(18) List any new IANA registries that require Expert Review for future
+allocations. Provide any public guidance that the IESG would find useful in
+selecting the IANA Experts for these new registries.
+-->
+
+The document set defines several IANA registries that offer “Provisional
+Registrations” and require Expert review. The IESG should select subject matter
```
Comment (MikeBishop) [inline writeups/base-drafts.md:284]:
```suggestion
The document set defines several IANA registries that offer "Provisional
Registrations" and require Expert review. The IESG should select subject matter
```
I question the use of smart quotes.  Same in @LPardue's suggestion.

Comment (larseggert) [reply writeups/base-drafts.md]: Will fix.

**Comment by larseggert** [comment]
I'm merging this now. We can continue to word-smith on the merged version, if needed.

**Pull request merged**